apps/trading-e2e/src/integration/wallet-eth.cy.ts

```diff
@@ -0,0 +1,58 @@
+import { connectEthereumWallet } from '../support/ethereum-wallet';
+
+const connectEthWalletBtn = 'connect-eth-wallet-btn';
+
+describe('ethereum wallet', { tags: '@smoke' }, () => {
+  beforeEach(() => {
+    cy.mockWeb3Provider();
+    // Using portfolio withdrawals tab is it requires Ethereum wallet connection
+    cy.mockTradingPage();
+    cy.mockSubscription();
```

Comment:
Can the mocks be run in a before hook rather than beforeEach?

Reply:
Probably now because "All intercepts are automatically cleared before every test."
https://docs.cypress.io/api/commands/intercept